### ISSUE TITLE
className not propagated when inline=true

### ISSUE THREAD

**Issue by mnpenner** [rexxars/react-lowlight#3]
I was trying to add a `className` to some `inline` code, but it's not added to the `<code>` block like I'd expect.

Checking the source, I can see why:

```js
  var code = h('code', codeProps, value)
  return props.inline ? code : h('pre', {className: props.className}, code)
```

Could it be added please?

**Comment by rexxars** [comment]
Fixed in 2.0.0 (as well as upgraded all dependencies to latest versions)

**Comment by mnpenner** [comment]
Perfect. Works great, thank you!
